### PR TITLE
Create and publish symbols nupkg

### LINKED ISSUE #982: Potential issue with Source Link configuration
# Bug Report

## Prerequisites

- [x] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [x] Are you running the latest version of AngleSharp?
- [x] Did you check the FAQs to see if that helps you?
- [x] Are you reporting to the correct repository? (there are multiple AngleSharp libraries, e.g., `AngleSharp.Css` for CSS support)
- [x] Did you perform a search in the issues?

For more information, see the `CONTRIBUTING` guide.

## Description

Was having trouble stepping through some behaviour in my code that went through to AngleSharp. It doesn't seem like Source Link is configured correctly (at least according to the [NuGet Package Explorer app](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer)). This is what I see when opening the AngleSharp NuGet package in it:

image

I don't see anything obviously wrong in the `AngleSharp.Core.csproj` file, at least compared to my own libraries:

image

Not sure if you might have any insights into what is going on? I tried to see if I could get an artifact from the CI to check but it doesn't seem your CI builds store NuGet package artifacts.

### PR DESCRIPTION
**Pull request by campersau**
# Types of Changes

Create and publish snupkg.

## Prerequisites

Please make sure you can check the following two boxes:

- [X] I have read the **CONTRIBUTING** document
- [X] My code follows the code style of this project

## Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

## Description

Create and publish snupkg.

Example: https://www.myget.org/feed/campersau/package/nuget/AngleSharp/0.16.1-alpha-27
image

Fixes https://github.com/AngleSharp/AngleSharp/issues/982

### CODE REVIEW

File: tools/anglesharp.cake [hunk 1]
```diff
@@ -125,7 +126,8 @@ Task("Create-Package")
         {
             Version = version,
             OutputDirectory = nugetRoot,
-            Symbols = false,
+            Symbols = true,
+            ArgumentCustomization = args => args.Append("-SymbolPackageFormat snupkg"),
```
Comment (campersau) [inline tools/anglesharp.cake:130]: Workaround for https://github.com/cake-build/cake/issues/2362 since cake version 1.2.0 is not yet released.

Comment (FlorianRappl) [reply tools/anglesharp.cake:130]: Awesome - you're the man! 🍻

File: tools/anglesharp.cake [hunk 2]
```diff
@@ -41,7 +41,7 @@ Setup(_ =>
     Information($"Building version {version} of {projectName}.");
     Information("For the publish target the following environment variables need to be set:");
     Information("- NUGET_API_KEY");
-    Information("- GITHUB_API_TOKEN");
+    Information("- GITHUB_TOKEN");
```
Comment (FlorianRappl) [inline tools/anglesharp.cake:44]: Great catch!

**Review by FlorianRappl — APPROVED** [review]
Love it. Amazing contribution! So great to have you contributing @campersau !

🍻

File: tools/anglesharp.cake (hunk 2 above)
Comment (campersau) [reply tools/anglesharp.cake:44]: Yeah, I missed that the last time :)

**Comment by campersau** [comment]
Strange here it did not push the `snupkg` automatically:
https://github.com/AngleSharp/AngleSharp/runs/2859635799?check_suite_focus=true#step:3:409

In my runs it worked:
https://github.com/campersau/AngleSharp/runs/2857608276?check_suite_focus=true#step:3:413

**Comment by campersau** [comment]
Looks like we need to set the Source to `https://api.nuget.org/v3/index.json` to upload symbols. @FlorianRappl should I do another PR for it?

**Comment by FlorianRappl** [comment]
Would be superb - thanks for your efforts @campersau !

**Pull request merged**